internal/types/types.go

```diff
@@ -97,7 +97,10 @@ var Null = NullType{}
 func validateValue(value any) error {
 	switch value := value.(type) {
 	case *Document:
-		return value.validate()
+		// TODO Call new validation here: https://github.com/FerretDB/FerretDB/issues/693
+		// Attention! This place require us to know whether the doc is a data doc or command doc!
```

Comment:
@AlekSi, an example where we need to know if it's a data or command document.

Reply:
Let's remove all validation code, not just comment it out. Half of it is wrong, and another is in the wrong place. We could always get old code from the git history if we need it